### ISSUE TITLE
Processing Error

### ISSUE THREAD

**Issue by onionbunny** [pFindStudio/pGlyco3#27]
Hi,

I encountered a problem while running pGlyco3. 
How should I solve it?

Many thanks.
<img width="733" alt="20231217132948" src="https://github.com/pFindStudio/pGlyco3/assets/104182173/1e43e900-70c4-4573-984c-de7ccb7cfcc3">

**Comment by jalew188** [comment]
This is fine, this is just a warning exception in pGlycoSite. The results were generated in the output folder

**Comment by onionbunny** [comment]
Thank you！While recently encountered a new problem, why does pGlycoSite give an error? How could I make it？
<img width="731" alt="4a6a3f9c243ab0dae225b1725824480" src="https://github.com/pFindStudio/pGlyco3/assets/104182173/d89fd3e6-b6ce-47c1-976b-4bc9e3b559a0">

**Comment by jalew188** [comment]
Maybe your pGlyco version is expired, you need to download the new version at https://github.com/pFindStudio/pGlyco3/releases

**Comment by onionbunny** [comment]
Thank you, but I used the latest version, and this occurs when choose 'HCD+EThcD', as log below:
Process ID=37212: ??????????????:  System.IO.FileNotFoundException: ??????????????????????Base, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null????????????????????????????????????????????
Process ID=37212:    ?? CombineMS2.Program.Main(String[] args)
Process ID=41544: [pParse] <INFO> - Closed E:


I have tested several times, could I get some advice? Thanks!

**Comment by jalew188** [comment]
Ahh, it is bad allocaiton error from pGlycoDB kernel. Maybe you need to remove all other files except for RAW files. And then process the the raw data again.

**Comment by jalew188** [comment]
> Thank you, but I used the latest version, and this occurs when choose 'HCD+EThcD', as log below:
> 
> Process ID=38024: [pGlyco Error] Open in CMGFReader: Could not open file: E: …… Process ID=37212: ??????????????: System.IO.FileNotFoundException: ??????????????????????Base, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null???????????????????????????????????????????? Process ID=37212: ?? CombineMS2.Program.Main(String[] args) Process ID=41544: [pParse] - Closed E:
> 
> I have tested several times, could I get some advice? Thanks!

Maybe it is the issue of .NET version, mine is .NET framework 7.0.x. You can check by `dotnet --version`.

**Comment by onionbunny** [comment]
Yes, I checked .NET version, and it seemed not the reason of error. Could I got another advice? Thank you!
<img width="323" alt="19a06fb34afbf0172a94e84a4403a24" src="https://github.com/pFindStudio/pGlyco3/assets/104182173/69877c89-e632-420f-b07d-71d8363a4063">

**Comment by jalew188** [comment]
Try `pGlyco_setup.exe` in the release page?

**Comment by Drhrd** [comment]
Hej hej, any findings from the host? Got trapped the same way for 5 mths...  Really need your help.
And for the developer, pGlyco.Site.exe source code desired :). THX
